deepmd/pt/model/task/dipole.py

```diff
@@ -103,6 +104,7 @@ def __init__(
             resnet_dt=resnet_dt,
             numb_fparam=numb_fparam,
             numb_aparam=numb_aparam,
+            numb_caseid=numb_caseid,
```

Comment:
`numb_case` sounds more reasonable

Reply:
`numb_caseid` means the dimension of `caseid`, similar as `numb_fparam`.  `numb_caseid` == `numb_case` only when we use one-hot embedding, maybe we will use other methods in teh future. So I suggest to use `numb_caseid` here.

Reply:
Then it is more reasonable to directly set the encoded case, not the id of the case.

Reply:
By your currently design, you set the index of a case, number of cases is more explanatory than number of caseid

Reply:
Renamed to `dim_case_embd`.